data/gameMinterFees.ts

```diff
@@ -0,0 +1,2 @@
+export const gameMintingFee = 300;
+export const gameUpdateFee = 100;
```

Comment:
I'm not sure about the actual values we want to use here.

Reply:
Me neither, let's add a // TODO comment as I expect this not to be known straight away